dali/imgcodec/decoders/libjpeg_turbo.cc

```diff
@@ -21,11 +21,11 @@
 namespace dali {
 namespace imgcodec {
 
-DecodeResult LibJpegTurboDecoderInstance::Decode(DecodeContext ctx,
-                                                 SampleView<CPUBackend> out,
-                                                 ImageSource *in,
-                                                 DecodeParams opts,
-                                                 const ROI &roi) {
+DecodeResult LibJpegTurboDecoderInstance::DecodeImplTask(int thread_idx,
```

Comment:
This is the bugfix. This decoder only implemented the single image API (overriding the default). Since DecodeImpTask was not defined for any backend, the batched API resulted in an error. The fix is to define DecodeImplTask and use the base class Decode APIs for both single image and batch.